package.json

```diff
@@ -714,7 +714,6 @@
     "leaflet": "1.5.1",
     "leaflet-draw": "0.4.14",
     "leaflet-responsive-popup": "0.6.4",
-    "leaflet-vega": "^0.8.6",
```

Comment:
@thomasneirynck @nreese  Am I right that we don't need that dependency anymore

Reply:
Removing leaflet-vega should be fine since leaflet is no longer used in vega maps.